binary_search_tree/binary_search_tree.py

```diff
@@ -5,55 +5,30 @@ def __init__(self, value):
     self.right = None
 
   def insert(self, value):
-    # print('self.value', self.value, self.left, self.right, value)
-    print('self.value', self.value)
-    print('lefty', self.left)
-    print('right', self.right)
-    print(value)
-    parent = self.value
-    print(self)
-    if self.value is None:
-      self.value = value
-      print('1****')
+    if value < self.value:
+      if self.left is None:
+        self.left = BinarySearchTree(value)
+      else:
+        return self.left.insert(value)
     else:
-      if self.value < value:
-        if self.right is None:
-          self.right = value
-          print('R', self.value, self.left, self.right)
-          return BinarySearchTree.insert(self, value)
-          
-        else:
-          print('R ELSE', self.value, self.left, self.right)
-          return BinarySearchTree(self.right)
+      if self.right is None:
+        self.right = BinarySearchTree(value)
       else:
-        print(value, self)
-        if self.left is None:
-          self.left = value
-          print('L', self.value, self.left, self.right)
-          return BinarySearchTree.insert(self, value)
-          
-        else:
-          print('L ELSE', self.value, self.left, self.right)
-          return BinarySearchTree(self.left)
-   
+        return self.right.insert(value)
 
```

Comment:
Great job on completing BST! I like that you are using elif in your contains function